spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:each) do
+      get '/users/:user_id/posts'
+    end
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'return 200' do
+      expect(response.status).to eq(200)
+    end
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
```

Comment:
- [ ] Kindly add a test that checks If the response body includes the correct placeholder text for `post#index` action as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20the%20response%20body%20includes%20correct%20placeholder%20text.).